frontend/src/hooks/useSafeAuth.tsx

```diff
@@ -0,0 +1,40 @@
+import { User, UserManagerEvents } from 'oidc-client-ts';
+import { useContext } from 'react';
+import { AuthContext, AuthContextProps } from 'react-oidc-context';
+import { toast } from 'react-toastify';
+
+const MockAuthContext: AuthContextProps = {
+  settings: {
+    authority: 'https://localhost:5001',
+    client_id: 'test',
+    redirect_uri: 'https://localhost:3000',
```

Comment:
Is it guaranteed that they won't be used? Can't we use Undefined or similar here? Please add a bit more documentation how it works.

Reply:
I will change it to something that is not a valid config. `undefined` does not work because it requires strings.
> Is it guaranteed that they won't be used?

Yes, the `FallbackAuthContext` is not used by the `AuthProvider` inside `Providers.tsx`.
I will add a little more explanation.